### ISSUE TITLE
Develop a list of accessibility evaluation rules based on the aria spec and aria authoring practices

### ISSUE THREAD

**Issue by jongund** [w3c/aria#670]
The WCAG working group is developing a evaluation rule format specification and working on collecting rules related to WCAG conformance.   There are ARIA design pattern and semantic rules that should be extracted from the ARIA documents and submitted to the WCAG 2.0 process.

**Comment by jnurthen** [comment]
Not sure what to do with this.

**Comment by jongund** [comment]
@jnurthen 
@wilcofiers

This might be something the new W3C community on accessibility rules could help work on, I would be interested in this project.

[ACT RULES COMMUNITY GROUP](https://www.w3.org/community/act-r/)

Jon
